_dashboards/drag-drop-wizard.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Drag and drop wizard (experimental feature)
+nav_order: 15
+---
+
+# Build visualizations by  dragging and dropping fields
+
+You can use the drag and drop visualization Wizard in OpenSearch Dashboards to create your data visualizations easily with just a few clicks or a single mouse gesture. 
```

Comment:
lowercase 'w' on wizard

Reply:
it's uppercase in the UI, so text should match it

Reply:
Name will change in release 2.4, so we'll update naming convention again